### PR TITLE
1292038: Changed adjustments to GtkAdjustment objects

### PR DESCRIPTION
**Pull request by Ceiu**

### CODE REVIEW

File: src/subscription_manager/gui/data/glade/installed.glade [hunk 1]
```diff
@@ -3,6 +3,19 @@
   <!-- interface-requires gtk+ 2.10 -->
   <!--
  interface-naming-policy project-wide -->
+  <object class="GtkAdjustment" id="adjustment1">
+    <property name="upper">396</property>
+    <property name="step_increment">39.6</property>
+    <property name="page_increment">356.4</property>
```
Comment (alikins) [inline src/subscription_manager/gui/data/glade/installed.glade]: As best I can tell, the scrolled windows don't actually seem to care what these values are, just that they have an Adjustment attached.

Something like
``` xml
<object class="GtkAdjustment" id="adjustment1">
    <property name="upper">100</property>
    <property name="step_increment">1</property>
    <property name="page_increment">10</property>
 </object>
```
... should work just as well and be a a little less confusing. I think the number in the old glade versions just
correspond to whatever glade thought the adjustment values were when the project was saved.

File: src/subscription_manager/gui/data/glade/installed.glade [hunk 2]
```diff
@@ -3,6 +3,19 @@
...
+    <property name="page_size">396</property>
+  </object>
+  <object class="GtkAdjustment" id="adjustment2">
```
Comment (alikins) [inline src/subscription_manager/gui/data/glade/installed.glade]: adjustment1 et al is the default glade naming scheme, maybe something like 'scrolled_window_3_h_adjustment' and 'scrolled_window_3_v_adjustment'?  (ala, gui/data/ui/installed.ui)

Comment (Ceiu) [inline src/subscription_manager/gui/data/glade/installed.glade]: Done.

**Comment by wottop** [comment]
ack

**Pull request merged**